policy/ibackend.go

```diff
@@ -9,7 +9,7 @@ import (
 type IBackend interface {
 	Init(cfg config.Store) error
 	GetName() string
-	Evaluate(
+	BackEndEvaluate(
```

Comment:
why do we need to change this function name?

Reply:
I thought it is better to have different interface name in the same package.

Reply:
I'd rather `ibackend::evaluate` than `ibackend::backend_evaluate`, no?

Reply:
OK, I am not too fussy about names, I can revert it back to Evaluate, if that is more convenient!